src/vhdlConvertor/packageParser.cpp

```diff
@@ -39,6 +39,7 @@ std::unique_ptr<HdlNamespace> VhdlPackageParser::visitPackage_body(
 	// ;
 	p = create_object<HdlNamespace>(ctx);
 	p->name = VhdlLiteralParser::getIdentifierStr(ctx->identifier(0));
+        p->defs_only = false;
```

Comment:
I know this is the default.  Personally, I like for it to be explicit.  Making it explicit removes the connascence between the parser and the hdlObject relative to the default value.  It also makes the `VhdlPackageParser` more parallel to the `VhdlPackageHeaderParser`.